webapp/templates/public/homepage.html.twig

```diff
@@ -65,5 +66,11 @@
                 </div>
             </div>
         </div>
+
+        <div class="row d-flex flex-column flex-lg-row">
+            {% for post in blogPosts %}
+                {% include 'public/partials/blog_post_on_homepage.html.twig' %}
```

Comment:
Tady je to takove zvlastni.
Sice to bude fungovat, protoze se promenne jmenuji stejne v parent i child sablone.
Ale obecne bych to nedoporucoval, radsi pouzivat with {'key': 'val'}

[docs](https://twig.symfony.com/doc/3.x/tags/include.html#:~:text=%7B%25%20include%20%27template.html%27%20with%20%7B%27foo%27%3A%20%27bar%27%7D%20%25%7D)